sway/commands/blur.c

```diff
@@ -12,7 +16,7 @@ struct cmd_results *cmd_blur(int argc, char **argv) {
 
 	struct sway_container *con = config->handler_context.container;
 
-	bool result = parse_boolean(argv[0], config->blur_enabled);
+	bool result = cmd_blur_parse_value(argv[0]);
```

Comment:
What's the reasoning for this change?

Reply:
Yeah I think we should keep the one liners as is. Good thinking with the corner radius though!